backend/ShoulderToShoulder/s2s/db_models/event.py

```diff
@@ -0,0 +1,12 @@
+from django.db import models
+
+
+class Event(models.Model):
+    title = models.CharField(max_length=100)
+    date = models.DateField()
+    location = models.CharField(max_length=200)
+    attendees = models.ManyToManyField('User')
+    
+    def __str__(self) -> str:
```

Comment:
Make sure to double check that the __str__ function is correct; right now, it is trying to print the event model using `self.description`, but `self.description` doesn't exist in this model.